test/integration/cloudresource_e2e_test.go

```diff
@@ -113,12 +114,18 @@ var _ = Describe(fmt.Sprintf("%s,%s: NetworkPolicy On Cloud Resources", focusAws
 	})
 
 	configANPApplyTo = func(kind, instanceName, vpc, tagKey, tagVal string) *k8stemplates.EntitySelectorParameters {
-		ret := &k8stemplates.EntitySelectorParameters{
-			Kind:              strings.ToLower(kind),
-			VPC:               strings.ToLower(vpc),
-			CloudInstanceName: strings.ToLower(instanceName),
+		ret := &k8stemplates.EntitySelectorParameters{}
+		if len(kind) > 0 {
```

Comment:
do you really need len check?

Reply:
Its the most straightforward way. Another possible way I found is https://stackoverflow.com/questions/18276173/calling-a-template-with-several-pipeline-parameters/18276968#18276968, but I think its just unnecessarily compplicated.